README.md

```diff
@@ -431,3 +431,13 @@ When no value is set for a requested path, this method returns `nil`:
 
 Please visit [sensuapp.org/support](http://sensuapp.org/support) for details on community and commercial
 support resources, including the official IRC channel.
+
+## Build and Release
+
+For maintainers looking to release new versions of this cookbook you should follow this process:
+1. Add any `README.md` and `CHANGELOG.md` changes with links to Pull Requests. Commit this to develop branch.
+1. Create a commit to then tag for release I would suggest something like this `git commit -am 'prep for v$MAJOR.$MINOR.$RELEASE release'`. Commit this to develop and make sure that everything is good to go (ci passing and such).
```

Comment:
Missing the step with updating the `CHANGELOG.md` for versions and diff links. Will add and self merge.